### DISCUSSION TITLE
Keyboard Mode Free Loop Length

### DISCUSSION THREAD

**Discussion by Notoriouspud1** [SynthstromAudible/DelugeFirmware#1735] [Ideas]
It would be great to be able to live loop in keyboard mode without setting the length before recording. I would like to be able to press record in keyboard mode and have the loop start at the beginning of the next bar, and then press record again to have it stop recording at the end of the current bar. 

Seems easy enough, but I’m no coder, so it would be great to hear if this would be possible.

**Comment by racecube** [comment]
this would be a nice feature- like doubling clip length in keyboard view which i asked on another place before! 
Hope someone of the coders puts it on his list
